components/tcf/ui/src/FirstLayer/index.js

```diff
@@ -37,7 +31,7 @@ export default function TcfFirstLayer({
         handleCookiePolicyLayerClick
       )
     onOpenCookiePolicyLayer()
-  })
+  },[])
```

Comment:
👀

Reply:
I revert this. useCallback needs to be removed